SessionMessagingKit/Open Groups/OpenGroupAPI.swift

```diff
@@ -326,20 +329,25 @@ public enum OpenGroupAPI {
     /// methods for the documented behaviour of each method
     public static func preparedCapabilitiesAndRooms(
         authMethod: AuthenticationMethod,
-        using dependencies: Dependencies
+        using dependencies: Dependencies,
+        skipAuthentication: Bool = false
```

Comment:
`using dependencies` should always be the last parameter - please put `skipAuthentication` before it